<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SecurityTransactionDialog</class>
 <widget class="QDialog" name="SecurityTransactionDialog">
  <property name="windowModality">
   <enum>Qt::WindowModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>248</width>
    <height>309</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="locale">
   <locale language="English" country="UnitedKingdom"/>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QFormLayout" name="formLayout">
     <item row="0" column="0">
      <widget class="QLabel" name="typeLabel">
       <property name="text">
        <string>Type</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <layout class="QHBoxLayout" name="radioButtonHorizontalLayout">
       <item>
        <widget class="QRadioButton" name="buyRadioButton">
         <property name="text">
          <string>Buy</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QRadioButton" name="sellRadioButton">
         <property name="text">
          <string>Sell</string>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="horizontalSpacer">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="securityLabel">
       <property name="text">
        <string>Security</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QComboBox" name="securityComboBox"/>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="cashAccountLabel">
       <property name="text">
        <string>Cash Account</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <layout class="QHBoxLayout" name="cashAccountHorizontalLayout">
       <item>
        <widget class="QComboBox" name="cashAccountComboBox">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="editable">
          <bool>false</bool>
         </property>
         <property name="insertPolicy">
          <enum>QComboBox::NoInsert</enum>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QToolButton" name="cashAccountToolButton">
         <property name="text">
          <string>...</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item row="3" column="0">
      <widget class="QLabel" name="securityAccountLabel">
       <property name="text">
        <string>Security Account</string>
       </property>
      </widget>
     </item>
     <item row="3" column="1">
      <layout class="QHBoxLayout" name="securityAccountHorizontalLayout">
       <item>
        <widget class="QComboBox" name="securityAccountComboBox"/>
       </item>
       <item>
        <widget class="QToolButton" name="securityAccountToolButton">
         <property name="text">
          <string>...</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item row="4" column="0">
      <widget class="QLabel" name="dateLabel">
       <property name="text">
        <string>Date</string>
       </property>
      </widget>
     </item>
     <item row="4" column="1">
      <widget class="QDateEdit" name="dateEdit">
       <property name="calendarPopup">
        <bool>true</bool>
       </property>
       <property name="timeSpec">
        <enum>Qt::LocalTime</enum>
       </property>
      </widget>
     </item>
     <item row="5" column="0">
      <layout class="QVBoxLayout" name="descriptionLabelVerticalLayout">
       <item>
        <spacer name="descriptionLabelTopVerticalSpacer">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeType">
          <enum>QSizePolicy::Fixed</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>5</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QLabel" name="descriptionLabel">
         <property name="text">
          <string>Description</string>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="descriptionLabelBottomVerticalSpacer">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </item>
     <item row="5" column="1">
      <widget class="QPlainTextEdit" name="descriptionPlainTextEdit">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>30</height>
        </size>
       </property>
       <property name="tabChangesFocus">
        <bool>true</bool>
       </property>
       <property name="placeholderText">
        <string>Enter optional description</string>
       </property>
      </widget>
     </item>
     <item row="6" column="0">
      <widget class="QLabel" name="sharesLabel">
       <property name="text">
        <string>Shares</string>
       </property>
      </widget>
     </item>
     <item row="6" column="1">
      <widget class="QDoubleSpinBox" name="sharesDoubleSpinBox">
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
       <property name="showGroupSeparator" stdset="0">
        <bool>true</bool>
       </property>
       <property name="maximum">
        <double>10000000000000000.000000000000000</double>
       </property>
      </widget>
     </item>
     <item row="7" column="0">
      <widget class="QLabel" name="priceLabel">
       <property name="text">
        <string>Price per share</string>
       </property>
      </widget>
     </item>
     <item row="7" column="1">
      <widget class="QDoubleSpinBox" name="priceDoubleSpinBox">
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
       <property name="showGroupSeparator" stdset="0">
        <bool>true</bool>
       </property>
       <property name="decimals">
        <number>9</number>
       </property>
       <property name="maximum">
        <double>10000000000000000.000000000000000</double>
       </property>
      </widget>
     </item>
     <item row="8" column="0">
      <widget class="QLabel" name="totalLabel">
       <property name="text">
        <string>Total</string>
       </property>
      </widget>
     </item>
     <item row="8" column="1">
      <widget class="QDoubleSpinBox" name="totalDoubleSpinBox">
       <property name="enabled">
        <bool>true</bool>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
       <property name="readOnly">
        <bool>false</bool>
       </property>
       <property name="buttonSymbols">
        <enum>QAbstractSpinBox::UpDownArrows</enum>
       </property>
       <property name="showGroupSeparator" stdset="0">
        <bool>true</bool>
       </property>
       <property name="decimals">
        <number>9</number>
       </property>
       <property name="maximum">
        <double>10000000000000000.000000000000000</double>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="standardButtons">
      <set>QDialogButtonBox::NoButton</set>
     </property>
    </widget>
   </item>
  </layout>
  <action name="actionSelect_Cash_Account">
   <property name="text">
    <string>Select Cash Account</string>
   </property>
  </action>
  <action name="actionSelect_Security_Account">
   <property name="text">
    <string>Select Security Account</string>
   </property>
  </action>
 </widget>
 <tabstops>
  <tabstop>buyRadioButton</tabstop>
  <tabstop>sellRadioButton</tabstop>
  <tabstop>securityComboBox</tabstop>
  <tabstop>cashAccountComboBox</tabstop>
  <tabstop>cashAccountToolButton</tabstop>
  <tabstop>securityAccountComboBox</tabstop>
  <tabstop>securityAccountToolButton</tabstop>
  <tabstop>dateEdit</tabstop>
  <tabstop>descriptionPlainTextEdit</tabstop>
  <tabstop>sharesDoubleSpinBox</tabstop>
  <tabstop>priceDoubleSpinBox</tabstop>
  <tabstop>totalDoubleSpinBox</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
